app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/exemptionandlosses/in_year_losses.scala.html

```diff
@@ -70,13 +70,6 @@
   }
 }
 
-@details = @{
```

Comment:
this template won't be seen by people on a further return or amend return journey. It should remain unchanged. Although `isAmend` is not required since it will always be `false`

Reply:
if you look at the ticket description, Shraddha is asking for this page to be changed.

In year losses page - (changes required for both further returns and amends)

Reply:
the further returns and amend version of the in-year losses page is in the template `further_return_in_year_losses.scala.html` which you've changed - this template is for first returns only